packages/api-headless-cms/src/plugins/context.ts

```diff
@@ -3,7 +3,7 @@ import { CmsContext } from "~/types";
 
 export default () => {
     return new ContextPlugin<CmsContext>(context => {
-        const locale = context.i18n.getCurrentLocale();
+        const locale = context.i18nContent.getLocale();
```

Comment:
What is the difference between these 2 lines of code?

Reply:
i18n.getCurrentLocale() does not get the selected locale in the UI. it always returns en-us

Reply:
We should open this as a separate issue and ask @doitadrian for input. This behavior of `i18n.getCurrentLocale()` sounds odd/broken.

Reply:
Discussed this over a call. The change here is correct.